### PR TITLE
added julia ubi8.5 build script

### PR DESCRIPTION
**Pull request by pranavpandit1**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: j/julia/julia_v1.7.2_ubi8.5.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#Build cmake from source
+cd $SRCDIR
+wget https://cmake.org/files/v3.11/cmake-3.11.4.tar.gz
+tar -zxvf cmake-3.11.4.tar.gz
+
+cd cmake-3.11.4
+./bootstrap --system-curl
+make
+make install
+export PATH=/usr/local/bin:$PATH
```
Comment (amitsadaphule) [inline j/julia/julia_v1.7.2_ubi8.5.sh:50]: cmake 3.x is readily available on UBI 8.x. So, we don't really need to build cmake.

File: j/julia/julia_v1.7.2_ubi8.5.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+SRCDIR=`pwd`
+DIRNAME="julia"
+
+#Build gettext
+wget  https://ftp.gnu.org/pub/gnu/gettext/gettext-0.21.tar.gz
+tar -zxvf gettext-0.21.tar.gz
+cd gettext-0.21
+./configure
+make
+make install
```
Comment (amitsadaphule) [inline j/julia/julia_v1.7.2_ubi8.5.sh:39]: I don't think gettext is a dependency for v1.7.2. Can you please confirm?

File: j/julia/julia_v1.7.2_ubi8.5.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+#currently julia works on power with the following commit(v1.7.2)
+JULIA_VERSION=bf534986350a991e4a1b29126de0342ffd76205e
```
Comment (amitsadaphule) [inline j/julia/julia_v1.7.2_ubi8.5.sh:24]: Can we not checkout the v1.7.2 tag instead?

File: j/julia/julia_v1.7.2_ubi8.5.sh [hunk 4]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : julia
+# Version       : v1.7.2 (master branch, commit:bf534986350a991e4a1b29126de0342ffd76205e)
+# Source repo   : https://github.com/JuliaLang/julia/releases/tag/v1.7.2
```
Comment (amitsadaphule) [inline j/julia/julia_v1.7.2_ubi8.5.sh:6]: Repo url should be: https://github.com/JuliaLang/julia

File: j/julia/julia_v1.7.2_ubi8.5.sh [hunk 5]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : julia
+# Version       : v1.7.2 (master branch, commit:bf534986350a991e4a1b29126de0342ffd76205e)
```
Comment (amitsadaphule) [inline j/julia/julia_v1.7.2_ubi8.5.sh:5]: `v1.7.2` should be enough. We can omit `(master branch, commit:bf534986350a991e4a1b29126de0342ffd76205e)`.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: j/julia/julia_v1.7.2_ubi8.5.sh (hunk 2 above)
Comment (pranavpandit1) [reply j/julia/julia_v1.7.2_ubi8.5.sh:39]: there is a module call for msgfmt(), which has dependency on gettext package.

**Review by amitsadaphule — APPROVED** [review]
LGTM

**Pull request merged**